src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,6 @@
 
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T> {
```

Comment:
lets add an upper bound, because with this you can create a MachhineProducer<String> and string is not a machine